client/my-sites/plugins/index.js

```diff
@@ -71,6 +76,16 @@ export default function () {
 
 	page(
 		'/plugins',
+		selectSiteIfLoggedIn,
+		siteSelection,
+		navigation,
+		browsePlugins,
```

Comment:
I think we can safely remove the `browsePlugins` part

Reply:
Done!